### PR TITLE
Docker deployment

### PR DESCRIPTION
**Pull request by ddjnw1yu**

### CODE REVIEW

File: config/google.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const googleCredentials = {
   clientId: process.env.GOOGLE_CLIENT_ID,
   clientSecret: process.env.GOOGLE_CLIENT_SECRET,
-  redirectUri: process.env.GOOGLE_REDIRECT_URI,
+  redirectUri: process.env.GOOGLE_REDIRECT_URL,
```
Comment (alan-wu) [inline config/google.js]: Can we revert this change? The obejct variable is redirectUri.

Comment (ddjnw1yu) [reply config/google.js]: > Can we revert this change? The variable in class is redirectUri.

OK, I thought it is a typo. `url -> uri` Will change it soon.

**Review by alan-wu — APPROVED** [review]
Looks good to me.
